Source/Shaders/AdjustTranslucentFS.glsl

```diff
@@ -11,14 +11,14 @@ void main()
 {
     if (texture2D(u_depthTexture, v_textureCoordinates).r < 1.0)
     {
-#ifdef MRT
+#ifdef GL_EXT_draw_buffers
```

Comment:
Couldn't this stay as `#ifdef MRT`?  If the extension is not supported, the JavaScript code concatenating the shader should not `#defined MRT`.